src/manual-edits/content/docs/new-relic-only/tech-writer-style-guide/article-templates/landing_page_template.mdx

```diff
@@ -0,0 +1,155 @@
+<Callout variant="tip">
+  
+  This landing page information does not apply to the docs home page.
+  
+</Callout>
+
+
+If you need to create a new landing page, you can either copy an existing landing page, or you can modify the sample landing page shown at the bottom. Let's look at the sections you need for each landing page.
+
+## Front matter
+
+When you insert the front matter, be sure to designate the `type` as `landingPage`. Here's an example:
+
+```
+---
+title: New Relic APM
+type: landingPage
+tags:
+translate:
+redirects:
+---
+```
+
+## Introduction section
+
+After you insert the front matter, insert the introduction, which typically just includes text and an image. If you need help with inserting images, see the example at the bottom.
+
+## Tiles
+
+Tiles are a series of boxes that contain the main subject areas for your product. You should just list these in order you want them to appear, and the cascading style sheet will render them across the page.
+
+Here's an example of a tile: 
+
+```
+<LandingPageTileGrid>
+  <LandingPageTile
+    title="INSERT_YOUR_BOX_TITLE_HERE."
+    icon="fe-INSERT_THE_ICON_NAME"
+  >
+
+Insert tile content here...
+
+</LandingPageTile>
+```
+For each tile, do the following:
+
+* Insert a value for `title` that explains the purpose of the category.
+* Insert a value for `icon` by prefixing one of the [Feather Icons](https://github.com/newrelic/gatsby-theme-newrelic/tree/develop/packages/gatsby-theme-newrelic#available-icons) with `fe-` so it is recognized (for example, fe-alert-triangle).
+* Between the `LandingPageTile` tags, insert text, such as a bullet list with links to product documentation.
+
+## Button for viewing all docs in the category
+
+After your tiles, you should have a single button offers to take users to all the documentation for that category. 
+
+Here's an example:
+
+```
+<ButtonLink
+  role="button"
+  to="INSERT_LINK_TO_DIRECTORY_WITH_ALL_THESE_DOCS"
+  variant="normal"
+>
+  View all INSERT_YOUR_CATEGORY_HERE docs
+</ButtonLink>
+```
+
+## Code sample
+
+Here is the APM landing page you could modify to suit your needs:
+
+```
+---
+title: faucibus metus bohar
+type: landingPage
+tags:
+translate:
+redirects:
+---
+
+Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aenean cursus congue mi. Sed at ipsum ante. Integer lacinia, dui vestibulum ornare aliquet, orci ligula posuere neque, quis lobortis lectus nibh a orci. Integer dignissim ante vitae purus pulvinar tristique. Suspendisse quis risus ac turpis consectetur convallis vitae sit amet mauris.
+
+Sed tristique [INSERT_A_LINK_NAME](INSERT_THE_PATH_TO_THE_LINK) dignissim velit eu blandit. Phasellus cursus faucibus iaculis. Praesent vitae lacus pulvinar, finibus justo ut, ornare eros. Aenean convallis condimentum luctus. Ut tristique risus lacus, eget faucibus metus vestibulum vitae. Aliquam erat volutpat.
+
+
+![INSERT_ALT_TEXT_OR_CAPTION_FOR_THE_IMAGE](./images/INSERT_IMAGE_FILE_NAME.png "INSERT_THE_TITLE_OF_THE_IMAGE")
```

Comment:
Heads up, there may be a small tweak to this after #586 is finished. The 2-column layout is not currently preserved on the new site, which we want to add back. Likely there will be an additional component that will wrap this image + text. I'll let you know when thats ready to go so this can be updated accordingly.

Reply:
@jerelmiller, I had lost track of this, so do you have any updates on this?

Reply:
Ah yes! This has been tweaked. The structure is now as such:


```
<LandingPageHero>
  <HeroContent>
    Paragraph(s) about the stuff in the landing page.
  </HeroContent>

    ![NR One: APM landing page](./images/nrone-apm-landing-073120.png "NR One: APM landing page")

  <figcaption>
    Optional caption explaining the image above
  </figcaption>
</LandingPageHero>
```

Reply:
Here are a couple examples for you:

https://github.com/newrelic/docs-website/blob/develop/src/content/docs/apm/index.mdx
https://github.com/newrelic/docs-website/blob/develop/src/content/docs/browser/index.mdx

Reply:
@jerelmiller, I added the <figcaption> tag, but I was confused by your surrounding tag <LandingPageHero> and <HeroContent>. Are these just dummy tags?

Reply:
@rhetoric101 no, these are the components you'll need to wrap the hero content with in order to get it to render correctly. The links above show you what this look like in practice. To be more explicit, in order to get that nice looking 2-column hero area at the top of a landing page, you'll need:

* A `LandingPageHero` component wrapping the entirety of the content
* A `HeroContent` component wrapping the text portion of the hero (the stuff that will show on the left side)
* An image (or video)
* An (optional) caption, which is wrapped by the `<figcaption>` component.

The example above shows you the exact component structure you need to pull this off. The links provided show you the real world examples on how to get this. Does this make sense?

Reply:
@jerelmiller, sorry, I should have looked at the code of that sample page as you suggested! I updated this landing page template document with information about the LandingPageHero section. Please confirm that both the hero section and the final example are accurate.

Here are two more questions:

* I inserted a tip about feather icons showing various places to get them. Is this accurate and is the reason we explain this because users need to request additional Feather icons that are not in the current theme?
* I inserted a line that says you can/should indent figcaption the same as the image. I haven't tested this, but is this true?

Reply:
> I inserted a tip about feather icons showing various places to get them. Is this accurate and is the reason we explain this because users need to request additional Feather icons that are not in the current theme?

This looks great! I think this will suffice for now

> I inserted a line that says you can/should indent figcaption the same as the image. I haven't tested this, but is this true?

Yep! You are exactly correct!

Reply:
Hero example looks great! Thanks for updating 🙂